.github/workflows/release.yml

```diff
@@ -0,0 +1,97 @@
+name: Release
+on:
+  push:
+    tags:
+      - '([8-9]|[1-9][0-9]).[0-9]+.[0-9]+'
```

Comment:
Not sure if this fancy syntax is better than an explicit extra line for builds higher than 9.x.x